appendices/devbook-guide/text.xml

```diff
@@ -40,137 +39,111 @@ tags used in a GuideXML document:
-
-&lt;author title="Author"&gt;
-  &lt;mail link="yourname@gentoo.org"&gt;Your Name&lt;/mail&gt;
-&lt;/author&gt;
-
-&lt;abstract&gt;
-This guide shows you how to compose web documentation using
-our new lightweight Gentoo GuideXML syntax.  This syntax is the official
-format for Gentoo web documentation, and this document itself was created
-using GuideXML.
-&lt;/abstract&gt;
-
-&lt;!-- The content of this document is licensed under the CC-BY-SA license --&gt;
-&lt;!-- See https://creativecommons.org/licenses/by-sa/3.0 --&gt;
-&lt;license version="3.0"/&gt;
-
-&lt;version&gt;1&lt;/version&gt;
-&lt;date&gt;2011-11-29&lt;/date&gt;
+&lt;guide self="quickstart/"&gt;
+&lt;chapter&gt;
+&lt;title&gt;Quickstart Ebuild Guide&lt;/title&gt;
 </codesample>
 
 <p>
 On the first lines, we see the requisite tag that identifies this as an XML
-document and specifies its DTD. The <c>&lt;!-- &#36;Header&#36; --&gt;</c> line
-will be automatically modified by the CVS server and helps to track revisions.
-Next, there's a <c>&lt;guide&gt;</c> tag <d/> the entire guide document is
-enclosed within a <c>&lt;guide&gt; &lt;/guide&gt;</c> pair.
-</p>
-
-<p>
-The <c>lang</c> attribute should be used to specify the language code of your
-document. It is used to format the date and insert strings like "<e>Note</e>",
-"<e>Content</e>", etc. in the specified language. The default is English.
-</p>
-
-<p>
-Next, there's a <c>&lt;title&gt;</c> tag, used to set the title for the entire
-guide document.
-</p>
-
-<p>
-Then, we come to the <c>&lt;author&gt;</c> tags, which contain information
-about the various authors of the document.  Each <c>&lt;author&gt;</c> tag
-allows for an optional <c>title</c> element, used to specify the author's
-relationship to the document (author, co-author, editor, etc.).  In this
-particular example, the authors' names are enclosed in another tag <d/> a
-<c>&lt;mail&gt;</c> tag, used to specify an email address for this particular
-person. The <c>&lt;mail&gt;</c> tag is optional and can be omitted, and at
-least one <c>&lt;author&gt;</c> element is required per guide document.
+document. Next, there's a <c>&lt;guide&gt;</c> tag <d/> the entire document is
+enclosed within a <c>&lt;guide&gt; &lt;/guide&gt;</c> pair. Its <c>self</c>
+attribute must point to the relative path of the document from the root node.
+An exception is the root node itself, which has <c>&lt;guide root="true"&gt;</c>
+instead.
 </p>
 
 <p>
-Next, we come to the <c>&lt;abstract&gt;</c>, <c>&lt;version&gt;</c> and
-<c>&lt;date&gt;</c> tags, used to specify a summary of the document, the
-current version number, and the current version date (in YYYY-MM-DD format)
-respectively. Dates that are invalid or not in the YYYY-MM-DD format will
-appear verbatim in the rendered document.
+Next, there is a <c>&lt;chapter&gt;</c> tag. Every document must have exactly
+one chapter. Its <c>&lt;title&gt;</c> is used to set the title for the entire
+document.
```

Comment:
Should we mention that the body of the title shall not contain any newlines? e.g. don't do:
```
<title>
Foo
</title>
```

Reply:
Done.